api/src/main/java/com/netflix/iceberg/DataFile.java

```diff
@@ -19,6 +19,7 @@
 
 package com.netflix.iceberg;
 
+import com.netflix.iceberg.encryption.EncryptionKeyMetadata;
```

Comment:
Nit: this isn't used. Not a blocker, but we should remove it eventually.

Reply:
@yifeih think we can address this in https://github.com/apache/incubator-iceberg/pull/107?